### PR TITLE
Add README

### LINKED ISSUE #114: Complete README for TERRA-REF trait data
As part of wrapping up the data I've curated from TERRA-REF along with related ticket #111, add trait metadata to current README.

### LINKED ISSUE #124: Incorporate David's suggestions into README pull request
Follow-up ticket to #114 - need to edit PR as per David's suggestions
- [x] add README template from UA Libraries
- [x] go through edits with David

### PR DESCRIPTION
**Pull request by MagicMilly**
Still need

- links to output data on CyVerse 
    - could probably also upload to repo in `data` > `processed` due to being small files
- to update trait data cleaning notebooks within repo and on CyVerse

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,230 @@
...
+    - Planting: 2016-06-17, Day 169
+    - Harvest: 2016-10-21, Day 295
+- Clemson University Pee Dee Research and Education Center, South Carolina
+    - Coordinates: 34.289, -79.737 
+    - Elevation: 42 meters
+    - Planting: 2014-05-06, Day 126
+    - Latest date in Clemson trait data: 2014-10-15, Day 288
+    
+### Licenses
+TERRA-REF data and other documents are licensed under CC-By. All software is licensed under MIT.
```
Comment (dlebauer) [inline README.md]: data from Season 4 and 6 contained in this repository are licensed under CC-0.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,230 @@
...
+
+### Contacts
+Please contact these individuals with any questions
```
Comment (dlebauer) [inline README.md]: Remove Todd and Nadia's name from this derivative dataset that they are not familiar with. 

Also, add an empty line below the heading `### Contacts`

File: README.md [hunk 3]
```diff
@@ -1,2 +1,230 @@
...
+
+### Citations
+
+#### Dryad Data 
+
+Maricopa Agricultural Center (MAC) data from the TERRA-REF project seasons four and six have been published with Dryad and can be downloaded [here](https://doi.org/10.5061/dryad.4b8gtht99).
+
+**LeBauer, David et al. (2020), Data From: TERRA-REF, An open reference data set from high resolution genomics, phenomics, and imaging sensors, v6, Dryad, Dataset, https://doi.org/10.5061/dryad.4b8gtht99**
+
+---
```
Comment (dlebauer) [inline README.md]: Data from KSU are unpublished.

Data from Clemson come from a separate citation that should be listed here

File: README.md (hunk 1 above)
Comment (dlebauer) [reply README.md]: I think "Licenses" Contacts and Citations can be combined under a header "Data Use" this section can be combined with

**Review by dlebauer — CHANGES_REQUESTED** [review]
There are a number of details to discuss ... lets follow up with a real-time review and revision

**Comment by dlebauer** [comment]
TODO: 

- [x] update links to updated csvs on CyVerse that have fewer columns and match what is on dryad
- [ ] update trait data description  
- [ ] update weather data to match how trait data is presented
- [ ] update the proj. org. 
  - point to where all data is org. on CyVerse
  - put anything small enough (notebooks, references, scripts) in github

**Comment by MagicMilly** [comment]
All notebooks, raw downloads, weather script, and derived datasets are on CyVerse in the GenoPhenoEnvo data folder. The notebooks are in Pull Request #122. The task to `update weather data to match how trait data is presented` may need clarification from David. Tag me in a comment if there are any questions or permission issues. There are some outdated notebooks on CyVerse that Tyson has permission to delete.

**Pull request merged**